dsl/src/test/resources/core-bulk-extension-model.xml

```diff
@@ -32,7 +30,7 @@
         <set-variable encoding="Attribute" mimeType="Attribute" value="Attribute" variableName="Attribute"/>
     </flow>
     <flow name="flowFor-async">
-        <async name="Attribute" maxConcurrency="10000">
```

Comment:
Were these order changes necessary?

Reply:
they are the result of enforcing the order enricher.